mysql-test/suite/galera/t/galera_auto_inc_off.test

```diff
@@ -0,0 +1,51 @@
+#
+# 1) Check that the values of auto-increment step and offset,
+#    which are set by the user, are saved and restored when user
+#    turning off the automatic control for these options.
+# 2) Check that these options are successfuly passed to new session.
+# 3) Check that the same logic works for the auto-increment parameters
+#    passed via command line.
+#
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+SET GLOBAL auto_increment_increment=10;
+SET GLOBAL auto_increment_offset=7;
+SET GLOBAL wsrep_auto_increment_control='OFF';
+
+SHOW GLOBAL VARIABLES LIKE '%auto_increment%';
+
+--let $galera_connection_name = node_1a
+--let $galera_server_number = 1
+--source include/galera_connect.inc
+
+--connection node_1a
+
+SHOW VARIABLES LIKE '%auto_increment%';
+
+--connection node_1
+
+SET GLOBAL wsrep_auto_increment_control='ON';
+
```

Comment:
Added tests for wsrep_auto_increment_control = ON using wsrep variables to check values of the auto increment parameters (for the current automatic control algorithm).